### ISSUE TITLE
prevent table names from keywords and reserved words

### ISSUE THREAD

**Issue by jchouix** [trongate/trongate-framework#165]
Hello,
if I want to create a module with the name "groups", I will have a problem because the word "groups" is reserved in the mysql database.
To resolve this problem, it would be desirable to surround the $target_tbl variable with backticks (`) in the Model.php file.
What do you think ?
Best regards.

**Comment by DaFa66** [comment]
Yes, this would be an improvement to the Model.php and also implemented in the Desktop app as it also hangs when creating apps with hyphens in the name - e.g. 'tg-app'.

**Comment by trongate** [comment]
Simon, please do remind me to look into this during a livestream.
